src/game.js

```diff
@@ -0,0 +1,29 @@
+const wf = require('fs');
+
+class Game {
+  static create (params) {
+    Object.assign(this, params)
+    this.hint = Array(this.guess.length).join('_').split('').join(' ');
+    this.leftAttempts = 5
```

Comment:
you should not use "this" in an statics, only in constructor or class methods, otherwise you are accessing to the global scope